subsys/bluetooth/services/ras/rreq/ras_rreq.c

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
can this file go?

Reply:
is it causing any issues ? We're trying to merge this in parts/stages

Reply:
Add the file when you add the contents of the file, it doesn't make sense to add an empty file then add the contents in a different PR

Reply:
The two files are coming shortly in separate PRs; this has been split for ease of review.
If I don't bring them in then I won't have anything to put in zephyr_library_sources_ifdef for now so it'll be two empty CMakeLists.txt files instead.

I don't have a strong preference, can remove them and have empty cmakelists for now too - in either case the next PRs will follow as soon as this has been merged.

Reply:
if the cmake files would be empty, they can go, and be added in the PR adding the file contents

Reply:
Removed ras_rreq.c and the corresponding cmake file